tests/e2e/tests/RetryUnavailable.test.ts

```diff
@@ -21,8 +21,6 @@ test.describe("Retry unavailable states", () => {
       timeout: 20_000,
     });
 
-    await setTimeout(1000);
```

Comment:
Had to remove this timeout, probably because since now the peer reconciliation doesn't send the content directly it takes more time.